src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/NotifyParties.java

```diff
@@ -49,8 +48,8 @@ public AboutToStartOrSubmitResponse<CaseData, State> midEvent(CaseDetails<CaseDa
     private boolean checkNull(CaseData data) {
         return null != data.getCicCase()
             && (null == data.getCicCase().getNotifyPartySubject() || data.getCicCase().getNotifyPartySubject().isEmpty())
-            && (null == data.getCicCase().getNotifyPartyApplicant() || data.getCicCase().getNotifyPartyApplicant().isEmpty())
-            && (null == data.getCicCase().getNotifyPartyRepresentative() || data.getCicCase().getNotifyPartyRepresentative().isEmpty());
+            && (null == data.getCicCase().getNotifyPartyRepresentative() || data.getCicCase().getNotifyPartyRepresentative().isEmpty())
+            && (null == data.getCicCase().getNotifyPartyRespondent() || data.getCicCase().getNotifyPartyRespondent().isEmpty());
```

Comment:
Use the apache StringUtils.isNotBlank (null, "" or whitespace) or isNotEmpty (null, ""), perhaps? Would make it more readable, too.

Reply:
they are not strings, they are enum sets

Reply:
Ah, OK, then I suggest you use CollectionUtils.isEmpty, which is null-safe.

Replace 

`&& (null == data.getCicCase().getNotifyPartyApplicant() || data.getCicCase().getNotifyPartyApplicant().isEmpty())`

With

`&& (CollectionUtils.isEmpty(data.getCicCase().getNotifyPartyApplicant()))`